hack/terraform/eks/variables.tf

```diff
@@ -0,0 +1,68 @@
+variable eks_iam_instance_profile_name {}
+variable eks_cluster_iam_role_name {}
+variable eks_key_pair_name {}
+
+variable eks_open_port_begin {
+  default = 22
+}
+
+variable eks_open_port_end {
+  default = 22
+}
+
+variable eks_worker_count {
+  default = 2
+}
+
+variable "region" {
+  default = "us-west-2"
+}
+
+variable "map_accounts" {
```

Comment:
does all the stuff below actually need to be checked-in?

Reply:
they are required by eks modules in main.tf.

Reply:
I don't think that answers my question. According to the terraform docs, `map_accounts` is optional. So why do you bother with `map_accounts                         = var.map_accounts` in main.tf, given that it doesn't look like you are making any use of `map_accounts`? OR is it actually required to bring up the cluster?

Reply:
i confess I don't know enough about terraform-aws-eks  to give a qualified answer. It is present in https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/examples/basic/main.tf, which this check in is based on. I did not feel there is a need to mess around with a working setup.

Reply:
Not a big fan of keeping stuff we don't need / use, but this is unlikely to be a maintenance burden so we can keep it for now.